src/Administration/Resources/app/administration/src/module/sw-sso-error/page/index/index.spec.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @sw-package after-sales
+ */
+
+import './index';
+
+import { mount } from '@vue/test-utils';
+import { CookieStorage } from "cookie-storage";
+
+const { Component } = Shopware;
+
+async function createWrapper(useDefaultLogin) {
+    const wrapper = mount(await wrapTestComponent('sw-sso-error-index', { sync: true }), {
+        global: {
+            provide: {
+                loginService: {
+                    getLoginTemplateConfig: () => {
+                        return new Promise((resolve) =>  {
+                            const response = {
+                                useDefault: useDefaultLogin,
+                                url: 'https://foo.bar.baz',
+                            }
+
+                            resolve(response);
+                        });
+                    },
+
+                    getStorage: () =>  {
+                        const storage = new CookieStorage();
+                        storage.setItem('user', 'foo@bar.baz');
+
+                        return storage;
+                    }
+                }
+            },
+        },
+    });
+
+    await flushPromises();
+
+    return wrapper;
+}
+
+describe('src/module/sw-sso-error/page/index', () => {
+    it('should be available', () => {
+        const components = Component.getComponentRegistry();
+        expect(components.has('sw-sso-error-index')).toBe(true);
+    });
+
+    it('should set the login config', async () => {
+        const wrapper = await createWrapper(false);
+        await flushPromises();
+
+        const config = wrapper.vm.loginConfig;
```

Comment:
Don't test component internals. Test the usage with public APIs like props, DOM, etc.